conda/build_pytorch.sh

```diff
@@ -353,10 +353,10 @@ for py_ver in "${DESIRED_PYTHON[@]}"; do
     # Build the package
     echo "Build $build_folder for Python version $py_ver"
     conda config --set anaconda_upload no
-    # There was a bug that was introduced in conda-package-handling >= 1.6.1 that makes archives
-    # above a certain size fail out when attempting to extract
-    # see: https://github.com/conda/conda-package-handling/issues/71
-    conda install -y "conda-package-handling=1.6.0"
+
+    conda env list
+    python --version
+    conda install -y "conda-package-handling=1.7.3"
```

Comment:
I think we can actually go ahead and just remove this pin altogether.

The upstream issue has been resolved